rosa_support/rosa-troubleshooting-deployments.adoc

```diff
@@ -3,10 +3,9 @@ include::_attributes/attributes-openshift-dedicated.adoc[]
 [id="rosa-troubleshooting-cluster-deployments"]
 = Troubleshooting cluster deployments
 :context: rosa-troubleshooting-cluster-deployments
+
 toc::[]
 
-[id="rosa-troubleshooting-deployment_{context}"]
-= Troubleshooting cluster deployments
 This document describes how to troubleshoot cluster deployment errors.
 
 include::modules/rosa-troubleshooting-general-deployment.adoc[leveloffset=+1]
```

Comment:
While out of scope for this PR, I do believe it is worth mentioning that this procedure module is (1) not formatted correctly; and (2) contains two distinct procedures. I understand you are routinely working through some mod docs improvements, so this is one for the backlog.